### PR TITLE
Updating grid topic

### PR DESCRIPTION
**Pull request by YoannaIvanova**

### CODE REVIEW

File: en/components/grid-grouping.md [hunk 1]
```diff
@@ -1,28 +1,24 @@
...
 
 ## GroupBy Row
 
-The Grid has a `GroupBy Row` symbol under `Features` that represents a special row in the Grid that is like a heading preceding the grouped records with some summary information about their amount within the group. Adding this symbol has only a visual effect on your design to make it more realistic, if you want to switch the GroupBy feature, insert and configure accordingly a `Group Area`.
-
-## Code generation
-
-If the `GrouBy Row` symbol is added make sure to place it within the group defining the Grid as a sibling to the Header and Body groups and to detach it in order to customize how grouping is applied.
+The Grid has a `GroupBy Row` symbol under `Features` that represents a special row in the Grid that is like a heading preceding the grouped records with some summary information about their amount within the group. Adding this symbol has only a visual effect on your design to make it more realistic if you want to switch the GroupBy feature, insert and configure accordingly a `Group Area`.
```
Comment (StefanIvanov) [inline en/components/grid-grouping.md:21]:
```suggestion
The Grid has a `GroupBy Row` symbol under `Features` that represents a special row in the Grid that is like a heading preceding the grouped records with some summary information about their amount within the group. Adding this symbol has only a visual effect on your design to make it more realistic. To switch the GroupBy feature on, insert and configure a `Group Area`.
```

File: en/components/grid-excel-style-filter.md [hunk 2]
```diff
@@ -31,22 +31,18 @@ The Excel Style Filter overlay lets you configure the Column name to reflect the
...
 
 Related topics:
 
 - [Grid](grid.md)
-- [Display Density](grid-display-density.md)
-- [Display Density](grid-sorting.md)
-- [Display Density](grid-column-moving.md)
-- [Display Density](grid-column-hiding.md)
-- [Display Density](grid-column-pinning.md)
+- [Grid Display Density](grid-display-density.md)
```
Comment (StefanIvanov) [inline en/components/grid-excel-style-filter.md:41]:
```suggestion
- [Grid Row Filter](grid-row-filter.md)
- [Grid Display Density](grid-display-density.md)
```

File: en/components/grid-summaries.md [hunk 3]
```diff
@@ -34,15 +32,6 @@ The Grid Summary Cell comes with basic styling flexibility through the various o
 
 <img class="responsive-img" src="../images/grid_summaries_styling.png" srcset="../images/grid_summaries_styling@2x.png 2x" />
```
Comment (StefanIvanov) [inline en/components/grid-summaries.md:33]: @YoannaIvanova Make a new image where both available and unavailable summary cells have the same background color. The current image looks broken.

File: en/components/grid-toolbar.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
 
 # Grid Toolbar
 
-Use the Grid Toolbar to define actions that affect the appearance and behavior of the whole grid. By default the Toolbar will contain actions for showing/hiding and pinning/unpinning of columns, as well as Exporting the Grid content in various formats. The Grid Toolbar is visually identical to the [Ignite UI for Angular Grid Toolbar Feature](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/toolbar.html)
+Use the Grid Toolbar to define actions that affect the appearance and behavior of the whole grid. By default, the Toolbar will contain actions for showing/hiding and pinning/unpinning of columns, as well as Exporting the Grid content in various formats. The Grid Toolbar is visually identical to the [Ignite UI for Angular Grid Toolbar Feature](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/toolbar.html)
 
 ## Grid Toolbar Demo
```
Comment (StefanIvanov) [inline en/components/grid-toolbar.md:12]: @YoannaIvanova I added one more section could you add an image that has a different layout and elements of the toolbar from the default configuration,

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

File: en/components/grid-skeleton.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
+---
+title: Skeleton Grid - Design System Component
+_description: The Skeleton Grid is a representation of a Grid component that is shown while data is being loaded in the background to provide content for its cells.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+# Skeleton Grid
+
+Use the Skeleton Grid Component to layout the same type of information that you would in the standard one, but to also indicate an application state when no data is present at the moment e.g. when data is loaded from a remote data source and there is either lack of connectivity, or it takes some time. The Skeleton Grid is just a visual representation of a [Grid](grid.md) with a known layout, but without any data to show.
```
Comment (StefanIvanov) [inline en/components/grid-skeleton.md:9]:
```suggestion
Use the Skeleton Grid to layout the same type of information that you would in the standard one, but to also indicate an application state when no data is present at the moment e.g. when data is loaded from a remote data source and there is either lack of connectivity, or it takes some time. The Skeleton Grid is just a visual representation of a [Grid](grid.md) with a known layout, but without any data to show.
```

File: en/components/grid-skeleton.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+The Skeleton Header, Body, and Summary Cells support the same three display density variants as the standard [Grid](grid.md): comfortable, cosy, and compact.
+
+<img class="responsive-img" src="../images/grid_skeleton_cell_header_comfortable.png" srcset="../images/grid_skeleton_cell_header_comfortable@2x.png 2x" />
+<img class="responsive-img" src="../images/grid_skeleton_cell_header_cosy.png" srcset="../images/grid_skeleton_cell_header_cosy@2x.png 2x" />
+<img class="responsive-img" src="../images/grid_skeleton_cell_header_compact.png" srcset="../images/grid_skeleton_cell_header_compact@2x.png 2x" />
+
+## Items and Features (Header Cell)
+
+The Skeleton Grid supports the same different feature configurations via the `Feature Left` and `Feature Right` overrides as the standard [Grid](grid.md).
```
Comment (StefanIvanov) [inline en/components/grid-skeleton.md:33]:
```suggestion
The Skeleton Grid supports the same feature configurations via the `Feature Left` and `Feature Right` overrides as the standard [Grid](grid.md).
```

**Review by StefanIvanov — APPROVED** [review]

**Comment by StefanIvanov** [comment]
@sdimchevski we are ready with the reviews of this set of topics so you may go ahead with the testing :)

**Review by sdimchevski — APPROVED** [review]

**Pull request merged**